Makefile

```diff
@@ -2026,6 +2029,19 @@ telemetry_deploy_cleanup: ## cleans up the service instance, Does not affect the
 	${CLEANUP_DIR_CMD} ${OPERATOR_BASE_DIR}/ceilometer-operator ${DEPLOY_DIR}
 	oc rsh -t $(DBSERVICE_CONTAINER) mysql -u root --password=${PASSWORD} -e "flush tables; drop database if exists aodh;" || true
 
+.PHONY: telemetry_kuttl_run
+telemetry_kuttl_run: ## runs kuttl tests for the telemetry operator, assumes that everything needed for running the test was deployed beforehand.
+	kubectl-kuttl test --config ${TELEMETRY_KUTTL_CONF} ${TELEMETRY_KUTTL_DIR} --namespace ${NAMESPACE}
+
+.PHONY: telemetry_kuttl
+telemetry_kuttl: export NAMESPACE = ${TELEMETRY_KUTTL_NAMESPACE}
+# Set the value of $TELEMETRY_KUTTL_NAMESPACE if you want to run the telemetry
+# kuttl tests in a namespace different than the default (telemetry-kuttl-tests)
+telemetry_kuttl: input deploy_cleanup telemetry telemetry_deploy_prep
```

Comment:
If Telemetry operator depends on other operators (such as MariaDB, Keystone, etc), then you'll need to address those dependencies here as pre-targets.  i.e. `kuttl_common_prep`.  For example [1].

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/c0a3767928bc188fcd9af004e977c55b6f2d251d/Makefile#L1611